deepchecks/vision/checks/distribution/train_test_label_drift.py

```diff
@@ -69,82 +104,35 @@ def run_logic(self, context: Context) -> CheckResult:
-
-            drift_score, method, display = calc_drift_and_plot(
-                train_distribution=train_label_distribution,
-                test_distribution=test_label_distribution,
-                plot_title=title,
-                column_type='categorical',
-            )
-
-            values_dict[title] = {'Drift score': drift_score, 'Method': method}
-            displays.append(display)
-
-            # Drift on bbox areas:
-            title = 'bbox area distribution'
-            train_label_distribution = histogram_in_batch(dataset=train_dataset, label_transformer=get_bbox_area)
-            test_label_distribution = histogram_in_batch(dataset=test_dataset, label_transformer=get_bbox_area)
-
-            drift_score, method, display = calc_drift_and_plot(
-                train_distribution=train_label_distribution,
-                test_distribution=test_label_distribution,
-                plot_title=title,
-                column_type='numerical'
-            )
-
-            values_dict[title] = {'Drift score': drift_score, 'Method': method}
-            displays.append(display)
+        # Drift on other custom metrics:
+        train_distributions, test_distributions = \
+            generate_label_histograms_by_batch(train_dataset=train_dataset, test_dataset=test_dataset,
+                                               label_transformers=label_transformers_list)
 
-            # Number of bboxes per image
-            title = 'Number of bboxes per image'
-            train_label_distribution = count_custom_transform_on_label(dataset=train_dataset,
-                                                                       label_transformer=count_num_bboxes)
-            test_label_distribution = count_custom_transform_on_label(dataset=test_dataset,
-                                                                      label_transformer=count_num_bboxes)
+        figs_configs = zip(label_transformers_list, train_distributions, test_distributions)
+        values_dict = {}
 
+        for d, train_label_distribution, test_label_distribution in figs_configs:
             drift_score, method, display = calc_drift_and_plot(
                 train_distribution=train_label_distribution,
                 test_distribution=test_label_distribution,
-                plot_title=title,
-                column_type='categorical',
+                plot_title=d['name'],
+                column_type='numerical' if d['is_continuous'] else 'categorical'
             )
 
-            values_dict[title] = {'Drift score': drift_score, 'Method': method}
+            values_dict[d['name']] = {'Drift score': drift_score, 'Method': method}
             displays.append(display)
 
-        else:
-            raise NotImplementedError('Currently not implemented')  # TODO
-
         headnote = """<span>
             The Drift score is a measure for the difference between two distributions, in this check - the test
```

Comment:
Also think that this text should be updated to explain what this "distribution" is.

Reply:
Done